packages/cozy-harvest-lib/test/components/AccountForm.spec.js

```diff
@@ -101,39 +101,71 @@ describe('AccountForm', () => {
     })
   })
 
-  it('should have disabled button if there is required field empty', () => {
-    const fields = {
-      test: {
-        type: 'text'
+  describe('Submit Button', () => {
+    const getButtonDisabledValue = wrapper =>
+      wrapper
+        .dive()
+        .find('DefaultButton')
+        .props().disabled
```

Comment:
Nit: I think getButtonDisabledValue() does a bit too much stuff (finding the button, and checking a prop), I find it a bit cumbersome to pass wrapper also.

I like to keep a global wrapper variable that helper function use directly.

It is a matter of taste and not blocking at all.

```
let wrapper

const findButton = () => wrapper.dive().find('DefaultButton').props().disabled

it('should do stuff', () => {
  wrapper = shallow(...)
  const btn = findButton()
  expect(btn.props().disabled).toBe(true)
})
```

Why is it necessary to `dive()` ?